tasks/rsync-deploy.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: install rsync
+  apt: name=rsync state=present
+  sudo: true
+- name: ensure destination folder
+  file: path={{ platform_client_base_path }}/server/www state=directory
+- name: deploy by sync
+  synchronize:
+  args:
+    archive: no
+    recursive: yes
+    perms: yes
+    times: yes
+    src: "{{ platform_client_deploy_src }}/"
+    dest: "{{ platform_client_base_path }}/server/www/"
+- name: copy config.js file to server
```

Comment:
Yea, although I'm thinking that the whole `server/www` thing doesn't really make sense in this kind of deployment.

Probably need to introduce better differentiation between the build folder and the deployment folder.